warcry_proj/.coveragerc

```diff
@@ -0,0 +1,3 @@
+[run]
```

Comment:
Rob, maybe just /examples/serverless-api-example/ this should likely be for any kind of APIs you want to bootstrap with a serverless Pyramid based system... not just auth?